src/main/java/com/cleanroommc/groovyscript/compat/vanilla/CraftingRecipe.java

```diff
@@ -155,14 +149,45 @@ public IIngredient getRecipeIngredient() {
         }
 
         public ItemStack getGivenInput() {
-            return givenInput;
+            // Copy mark from recipeIngredient to givenInput
+            ItemStackMixinExpansion copy = ItemStackMixinExpansion.of(givenInput);
```

Comment:
uh, this isnt actually a copy - `ItemStackMixinExpansion#of` is just `(ItemStackMixinExpansion) (Object) stack`. did you mean to run `ItemStackMixinExpansion#exactCopy`? is making a copy even needed here?

Reply:
>  is making a copy even needed here?

I don't think so.
Renamed it to `itemStack`.